src/Stryker.Core/Stryker.Core.UnitTest/Mutants/MutantOrchestratorTests.cs

```diff
@@ -886,6 +886,43 @@ public void ShouldMutatePropertiesAsArrowExpression()
             ShouldMutateSourceToExpected(source, expected);
         }
 
+
+        [Fact]
+        public void ShouldNotExhibitIssue1423()
```

Comment:
Could you maybe add a more descriptive name to this test? I would like a clearer name that allows me to see the problem at first glace. Maybe reference the issue in the description so when we need more info we can reference that.